src/content/docs/ru/tutorial/2-pages/1.mdx

```diff
@@ -0,0 +1,158 @@
+---
+type: tutorial
+title: Создайте свою первую страницу в Astro
+description: |-
+  Обучающий урок: Создание первого блога в Astro —
+  Добавление новых страниц на сайт с навигационными ссылками между ними
+i18nReady: true
+---
+import Checklist from '~/components/Checklist.astro';
+import Blanks from '~/components/tutorial/Blanks.astro';
+import Box from '~/components/tutorial/Box.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+
+Теперь, когда вы знаете, что файлы `.astro` отвечают за страницы на вашем вебсайте, давайте создадим одну!
+
+<PreCheck>
+  - Создайть две новые страницы на своем сайте: About и Blog
+  - Добавить навигационные ссылки на ваши страницы
+  - Отобразить обновленную версию своего сайта в Интернете
+</PreCheck>
+
+## Создание нового файла `.astro`
+
+1. Откройте панель файлов вашего редактора кода и перейдите в папку `src/pages/`, где вы увидите существующий файл `index.astro`.
+
+2. В той же папке создайте новый файл и назовите его `about.astro`.
+
+3. Скопируйте или перепишите содержимое файла `index.astro` в ваш новый файл `about.astro`.
+
+    :::tip
+    Ваш редактор может показывать круглую белую иконку на вкладке для этого файла. Это означает, что файл еще не сохранен. В меню Файл в VS Code включите «Автоматическое сохранение», и вам больше не нужно будет сохранять файлы вручную.
+    :::
+
+4. Добавьте `/about` в конце URL-адреса просмотра вашего сайта в строке адреса и проверьте, что вы можете загрузить страницу. (например, `http://localhost:3000/about`)
+
+Сейчас ваша страница «О сайте» должна выглядеть точно так же, как первая страница, но мы это изменим!
+
+## Редактирование вашей страницы
+
+Отредактируйте HTML-код, чтобы эта страница была о вас.
+
+Чтобы изменить или добавить больше контента на вашей странице «О сайте», добавьте дополнительные HTML-элементы, содержащие контент. Вы можете скопировать и вставить код HTML ниже между существующими тегами `<body></body>`, или создать свои.
+
+```astro title="src/pages/about.astro» ins={3-8} del={2}
+<body>
+  <h1>Мой сайт Astro</h1>
+  <h1>Обо мне</h1>
+  <h2>… и моём новом сайт на Astro!</h2>
+
+  <p>Я изучаю введение об Astro. Это вторая страница на моем сайте, и это первая, которую я построил самостоятельно!</p>
+
+  <p>Этот сайт будет обновляться по мере того, как я буду проходить больше уроков, поэтому продолжайте проверять его и следите за тем, как идет мой путь!</p>
+</body>
+```
+Теперь снова посетите свою страницу `/about` в вашей вкладке браузера, и вы должны увидеть ваш обновленный контент.
+
+## Добавление навигационных ссылок
+
+Чтобы было проще просматривать все ваши страницы, добавьте HTML-ссылки на навигационные страницы перед вашим `<h1>` (заголовок первого уровня) в верхней части обеих ваших страниц (`index.astro` и `about.astro`):
+
+```astro title="src/pages/about.astro» ins={1-2}
```

Comment:
```suggestion
```astro title="src/pages/about.astro" ins={1-2}
```

Reply:
Thanks this is my mistake with russian typography